### ISSUE TITLE
Cannot delete messages

### ISSUE THREAD

**Issue by CSab6482** [iandwelker/smserver#95]
I haven't tested deleting an entire conversation, but I can't delete individual messages, I either get a pop up or my phone goes into safe mode
![can't delete](https://user-images.githubusercontent.com/53708281/108615461-574e2e80-73d2-11eb-9c9c-76d0462bbd25.JPG)
[Springboard Delete Crash.txt](https://github.com/iandwelker/smserver/files/6017186/Springboard.Delete.Crash.txt)
[SMServer Delete Crash.txt](https://github.com/iandwelker/smserver/files/6017187/SMServer.Delete.Crash.txt)

**Comment by iandwelker** [comment]
Yup, I found the reason for this (as long as you're on iOS 13) &mdash; it'll be fixed with the next update

**Comment by iandwelker** [comment]
@CSab6482  could you test with version 0.7.4 to let me know if this is fixed? It should be but I can't be certain.

**Comment by CSab6482** [comment]
![nope](https://user-images.githubusercontent.com/53708281/112096886-7842a480-8b75-11eb-9e47-a47d969fb434.png)
^ unfortunately no, I just had this happen [iOS 14.4.1 (don't currently have cr4shed installed, will do that now)
